SECURITY.md

```diff
@@ -1,10 +1,10 @@
-# Reporting security issues
+# Security
 
 The Woodpecker maintainers take security seriously.  
 If you discover a security issue, please bring it to their attention right away!
 
 ### Reporting a Vulnerability
 
-Please **DO NOT** file a public issue, instead send your report privately to `security @ woodpecker-ci.org`.
+Please **DO NOT** file a public issue, instead send your report privately to [`security @ woodpecker-ci.org`](mailto:security@woodpecker-ci.org).
 
-Security reports are greatly appreciated and we will publicly thank you for it, although we keep your name confidential if you request it.
+Security reports are greatly appreciated, and we will publicly thank you for it, although we keep your name confidential if you request it.
```

Comment:
Security reports are greatly appreciated, and we will publicly thank you for it. If you choose to remain anonymous, we will respect your request and keep your name confidential.